src/Drivers/CmdLine/CmdLine.csproj

```diff
@@ -146,7 +146,6 @@
     <Environments Include="$(SolutionDir)Environments\ZX81\$(OutDir)Reko.Environments.ZX81.*" />
     <Libraries Include="$(SolutionDir)Libraries\Libc\$(OutDir)Reko.Libraries.Libc.*" />
     <Libraries Include="$(SolutionDir)Libraries\Python\$(OutDir)Reko.Libraries.Python.*" />
-    <ExternalComponents Include="$(SolutionDir)..\external\Capstone.NET\*.dll" />
```

Comment:
The missing <ExternalComponents /> element here is likely causing the build failure.